charts/pega/README.md

```diff
@@ -529,6 +529,25 @@ tier:
       serviceAccountName: MY_SERVICE_ACCOUNT_NAME
 ```
 
+### Sidecar Containers
+
+If the pod needs to run with one or more [sidecar containers](https://kubernetes.io/docs/concepts/workloads/pods/#how-pods-manage-multiple-containers), you can specify custom 'sidecarContainers' for your deployment tier. Each sidecar container must include a complete container definition, including a name, image, resources, etc.
+
+Example:
+
+```yaml
+tier:
```

Comment:
We're going to exclude it from the values.yaml itself because it would be a little messy to include it and it's not the typical expectation to have this.